frontend/packages/kubevirt-plugin/src/constants/vm/workload-profile.ts

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable lines-between-class-members */
+import { ObjectEnum } from '@console/shared/src/constants/object-enum';
+import { SelectDropdownObjectEnum } from '../select-dropdown-object-enum';
+import {
+  WORKLOAD_PROFILE_DESKTOP_DESC,
+  WORKLOAD_PROFILE_SERVER_DESC,
+  WORKLOAD_PROFILE_HIGH_PERFORMANCE_DESC,
+} from '../../utils/strings';
+
+export class WorkloadProfile extends SelectDropdownObjectEnum<string> {
+  static readonly DESKTOP = new WorkloadProfile('desktop', {
+    label: 'Desktop',
+    description: WORKLOAD_PROFILE_DESKTOP_DESC,
+    order: 1,
+  });
+  static readonly SERVER = new WorkloadProfile('server', {
+    label: 'Server',
+    isdefault: true,
```

Comment:
the defaults depend on the exact template (os) * and should not behard coded like this

* https://github.com/kubevirt/common-templates/blob/master/generate-templates.yaml